### ISSUE TITLE
Can't move .esp plugins in NMM 0.65.10 anymore

### ISSUE THREAD

**Issue by kg4nbx** [Nexus-Mods/Nexus-Mod-Manager#673]
About a week ago, I decided to scrap all my mods and start over with a new list since I upgraded a computer with a new graphics card.  I uninstalled Skyrim, wiped all the folders left behind manually to clean out all the mods I had.  Then I uninstalled NMM, deleted all the folders left behind manually, downloaded this version of the mod manager and then reinstalled and began the long process of installing new mods.  Everything was working fine at first, I could sort my order with LOOT and then reopen NMM and manually make any adjustments I needed.  Then a couple days ago it stopped letting me move plugins manually.  The arrows are grayed out and any attempts to drag/drop gives the circle with a line through it.  I tried to uninstall and reinstall but that didn't change anything.  Why would it work perfectly for a week and then just stop functioning?

**Comment by fujita600618** [comment]
Same problem here! and i refuse to use Vortex, installed it and did not like it and went back to NMM and discover same issue

**Comment by AstralPaladin** [comment]
Yeah I have the same issue for Fallout 4; the arrows are grayed out, and dragging the esp does nothing. I should note that it was working for a day after I updated from 0.65.2 (the one hosted on Nexus) to this version; but then it stopped working.
![moving load order not working 0 65 10](https://user-images.githubusercontent.com/46169766/50457335-430c6500-0920-11e9-869e-2add1e0555c9.png)
